src/app/components/cases/TableCases/TableCases.tsx

```diff
@@ -31,9 +19,9 @@ const TableCases: React.FC<Props> = ({ data, isBusy }) => {
       loading={ isBusy }
       numLoadingRows={ 10 }
       columns={ columns }
-      data={ values }
+      data={ data?.results || [] }
```

Comment:
Data is causing Typescript errors: Type 'Case[]' is not assignable to type 'ValueNodes[]'.
  Type 'Case' is not assignable to type 'ValueNodes'.
    Index signature for type 'string' is missing in type 'Case'.